src/components/common/OldMap.js

```diff
@@ -0,0 +1,722 @@
+                            />{' '}
+                            Chemical
+                          </div>
+                        }
+                      />
+                      <FormControlLabel
+                        control={
+                          <Checkbox
+                            checked={state.EnergyDevices}
+                            onChange={handleTypeChange}
+                            name="EnergyDevices"
+                            color="primary"
+                          />
+                        }
+                        label={
+                          <div>
+                            <i
+                              class="fa fa-map-marker"
+                              style={{ color: 'Magenta' }}
+                            />{' '}
+                            Energy devices
+                          </div>
+                        }
+                      />
+                      <FormControlLabel
+                        control={
+                          <Checkbox
+                            checked={state.Miscellaneous}
+                            onChange={handleTypeChange}
+                            name="Miscellaneous"
+                            color="primary"
+                          />
+                        }
+                        label={
+                          <div>
+                            <i
+                              class="fa fa-map-marker"
+                              style={{ color: 'LightSlateGrey' }}
+                            />{' '}
+                            Miscellaneous
+                          </div>
+                        }
+                      />
+                      <FormControlLabel
+                        control={
+                          <Checkbox
+                            checked={state.Other}
+                            onChange={handleTypeChange}
+                            name="Other"
+                            color="primary"
+                          />
+                        }
+                        label={
+                          <div>
+                            <i
+                              class="fa fa-map-marker"
+                              style={{ color: 'Red' }}
+                            />{' '}
+                            Other
+                          </div>
+                        }
+                      />
+                    </FormGroup>
+                  </label>
+                </Menu>
+              </React.Fragment>
+            )}
+          </PopupState>
+        </div>
+        <h1 className="map-title">Find Excessive Force Incidents</h1>
+      </div>
+      <div className="container">
+        <div className="map_box">
+          <ReactMapGL
+            {...viewport}
+            mapboxApiAccessToken={process.env.REACT_APP_MAPBOX_TOKEN}
+            mapStyle="mapbox://styles/martaalicja1/ckfx37li7117819paqxl1cp6x"
```

Comment:
Is this a URL you want public? I'm not sure. It could be something you throw into a config file or a .env file if it needs to be protect.